src/main.rs

```diff
@@ -57,9 +58,15 @@ fn main() -> Result<()> {
             let incremental = is_beator && args.is_present("incremental-opt");
             let prune = !is_beator || args.is_present("prune-model");
 
-            let program = load_object_file(&input)?;
+            let mut model;
+
+            if is_beator && !args.is_present("from-btor2") {
```

Comment:
This seems to always try to parse a BTOR2 file when the `qubot` command is given. Is this intended? I think we should change this to just `if !args.is_present("from-btor2") {...}` here instead

Reply:
yes you are right! thanks!